packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -100,6 +100,11 @@
   &:invalid {
     box-shadow: none;
   }
+
+  // Keep Chrome autofill behind the notched outline
+  &:-webkit-autofill {
+    z-index: auto !important;
```

Comment:
Is `!important` strictly necessary?

Reply:
Yes, it is necessary to add `!important` here since all `-webkit-autofill` default styles already have `!important` and it wouldn't work without this.